package.json

```diff
@@ -63,19 +62,18 @@
     "avsc": "^5.4.10",
     "base64-js": "^1.2.1",
     "cordova-android": "^7.1.4",
-    "cordova-media-with-compression": "^2.1.2",
     "cordova-android-support-gradle-release": "^3.0.0",
     "cordova-lib": "^9.0.1",
+    "cordova-media-with-compression": "^2.1.2",
     "cordova-plugin-android-permissions": "^1.0.0",
     "cordova-plugin-app-event": "git+https://github.com/katzer/cordova-plugin-app-event.git",
     "cordova-plugin-app-version": "^0.1.9",
     "cordova-plugin-badge": "^0.8.8",
     "cordova-plugin-camera": "^4.0.3",
     "cordova-plugin-device": "^2.0.2",
     "cordova-plugin-dialogs": "^2.0.1",
-    "cordova-plugin-fcm": "git+https://github.com/yatharthranjan/cordova-plugin-fcm.git",
     "cordova-plugin-file": "^6.0.1",
-    "cordova-plugin-firebase-analytics": "^1.0.1",
+    "cordova-plugin-firebase": "git+https://github.com/mpgxvii/cordova-plugin-firebase.git#upstream",
```

Comment:
another question. Sorry for coming back to this again. So we don't need fcm plugin anymore? What is the alternative we have implemented?
About the `firebase plugin`, I would really recommend publishing the cordova-plugin-firebase and use the version based dependency management. Same goes to the app-event plugin.

Reply:
https://www.jfrog.com/confluence/display/BT/npm+Repositories IF this is helpful for publishing the version on bintray

Reply:
Yup, good idea. I've published the plugin.